src/main/scala/beam/agentsim/agents/PersonAgent.scala

```diff
@@ -1380,7 +1380,7 @@ class PersonAgent(
           data.failedTrips.foreach(uncompletedTrip =>
             generateSkimData(tick, uncompletedTrip, failedTrip = true, currentActivityIndex, nextActivity(data))
           )
-          generateSkimData(tick, data.currentTrip.get, failedTrip = false, currentActivityIndex, nextActivity(data))
+          val correctedTrip = correctTripEndTime(data.currentTrip.get, tick, body.id, body.beamVehicleType.id)
```

Comment:
We don't use this `correctedTrip`, so it could be deleted.

Reply:
Deleted